apollo-router/src/plugins/telemetry/mod.rs

```diff
@@ -1233,17 +1183,19 @@ impl Telemetry {
                     );
                 }
 
-                metrics.http_requests_total.add(
-                    &opentelemetry::Context::current(),
+                u64_counter!(
+                    "apollo_router_http_requests_total",
+                    "Total number of HTTP requests made.",
                     1,
-                    &metric_attrs,
+                    metric_attrs
                 );
             }
         }
-        metrics.http_requests_duration.record(
-            &opentelemetry::Context::current(),
+        f64_histogram!(
+            "apollo_router_http_request_duration_seconds",
```

Comment:
Same comment about possible rename.

Reply:
It's not a rename. The original code is referencing `metrics.http_requests_duration` which is declared as: `apollo_router_http_request_duration_seconds`.